### DISCUSSION TITLE
The issue of generating XML files for doxygen prompted by the automatic compilation of readthedoc

### DISCUSSION THREAD

**Discussion by ziheng0** [sphinx-doc/sphinx#13142] [General]
Hello,

I have already generated the work locally, but when uploading project to the readthedoc, there is a prompt
FileNotFoundError: [Errno 2] No such file or directory: '/home/docs/checkouts/readthedocs.org/user_builds/quecopen/checkouts/latest/docs/source/../build/doxygen/xml/index.xml'

This may be because the output item for doxygen was not configured in ReadThedoc. How can I tell ReadThedoc to configure it correctly, just like using doxygen Doxyfile locally

Has anyone encountered this kind of problem before, 
Thank you in advance

**Comment by ziheng0** [comment]
solved

**Comment by kaycebasques** [comment]
@ziheng0 what was your solution?
